src/deploy/functions/checkRuntimeDependencies.ts

```diff
@@ -10,28 +9,13 @@ import { FirebaseError } from "../../error";
 const FAQ_URL = "https://firebase.google.com/support/faq#functions-runtime";
 const CLOUD_BUILD_API = "cloudbuild.googleapis.com";
 
-const DEFAULT_WARN_AFTER = 1588636800000; // 2020-05-05T00:00:00.000Z
-const DEFAULT_ERROR_AFTER = 1591315200000; // 2020-06-05T00:00:00.000Z
-
 function node8DeprecationWarning(): void {
   track("functions_runtime_notices", "nodejs8_deprecation_warning");
   logger.warn();
   logLabeledWarning(
     "functions",
     `The Node.js 8 runtime is deprecated and will be decommissioned on ${bold(
-      "2020-12-05"
-    )}. For more information, see: ${FAQ_URL}`
-  );
-  logger.warn();
-}
-
-function node10BillingWarning(errorAfter: number): void {
-  track("functions_runtime_notices", "nodejs10_billing_warning");
-  logger.warn();
-  logLabeledWarning(
-    "functions",
-    `Cloud Functions will soon require the pay-as-you-go (Blaze) billing plan to deploy. To avoid service disruption, upgrade before ${bold(
-      new Date(errorAfter).toISOString().substr(0, 10)
+      "2021-03-15"
```

Comment:
should change the sentence above to "was decomissioned'

Reply:
Node 8 has been deprecated, it hasn't yet been decommissioned. 2021-03-15 is in the future still 😄